pkgs/development/tools/hover/default.nix

```diff
@@ -0,0 +1,69 @@
+{ buildGoModule, fetchFromGitHub, fetchurl, glfw, libGL, libX11, libXcursor, libXi,
+  libXinerama, libXrandr, libXxf86vm, makeWrapper, pkgconfig, stdenv, unzip}:
+
+buildGoModule rec {
+  pname = "hover";
+  version = "unstable-2020-01-23";
+
+  subPackages = [ "." ];
+
+  modSha256 = "019lg3s86965jyqfy83hb52y5rb6g91prsg3ckjxbcnca68lsdja";
+
+  src = fetchFromGitHub {
+    rev = "a6f11fe3c4946c9e7cce75360902bd79a077ccf4";
+    owner = "go-flutter-desktop";
+    repo = pname;
+    sha256 = "0pwdzd34zcbyspl7qiwfcrppr6zxp2zliqg9n8s6s3l6k7mffl9j";
+  };
+
+  libFlutter = fetchurl {
+    name = "libflutter_engine.zip";
+    url = if stdenv.isDarwin then "https://storage.googleapis.com/flutter_infra/flutter/051762798888ebea4666c5f9c9f4c0e4d3b3b481/darwin-x64/FlutterEmbedder.framework.zip"
+          else "https://storage.googleapis.com/flutter_infra/flutter/051762798888ebea4666c5f9c9f4c0e4d3b3b481/linux-x64/linux-x64-embedder";
+    sha256 = "15p6cdvi6rajjs6vv79y7vsrwwpg2kk9qlg14891wrmdn9nvpqb4";
```

Comment:
Both urls resolve to the dance sha256? I was expecting the build to fail on either Darwin or Linux.

Reply:
Good question :thinking:, I'll update the sha256 anyway

Reply:
It's really a different sha256, I just don't understand how bors build didn't fail 
![image](https://user-images.githubusercontent.com/7820865/74592495-c8f5de80-5000-11ea-902e-81f9c7a16679.png)

Reply:
@kalbasit I updated the sha256 anyway, thanks!

Reply:
ofborg probably had a cached version, the sha256 determines the uniqueness, not the url